packages/rocketchat-api/server/v1/groups.js

```diff
@@ -1,10 +1,15 @@
 //Returns the private group subscription IF found otherwise it will reutrn the failure of why it didn't. Check the `statusCode` property
-function findPrivateGroupById({ roomId, userId, checkedArchived = true }) {
-	if (!roomId || !roomId.trim()) {
+function findPrivateGroupByIdOrName({ roomId, roomName, userId, checkedArchived = true }) {
+	if ((!roomId || !roomId.trim()) && (!roomName || !roomName.trim())) {
 		return RocketChat.API.v1.failure('Body param "roomId" is required');
```

Comment:
I pulled the latest code and this line has been changed to :
`throw new Meteor.Error('error-roomid-param-not-provided', 'The parameter "roomId" is required');`